test/test_artifacts/scripts/run_autogluon_tests.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+AUTOGLUON_VERSION=$(micromamba list | grep autogluon | tr -s ' ' | cut -d ' ' -f 3)
+git checkout tags/v$AUTOGLUON_VERSION
+
+# Run autogluon quick start as end-to-end check
+jupyter nbconvert --execute --to python docs/tutorials/tabular/tabular-quick-start.ipynb
+jupyter nbconvert --execute --to python docs/tutorials/timeseries/forecasting-quick-start.ipynb
+
+# Detect gpu and run multimodal quick start if presented
+python -c "import torch; exit(0) if torch.cuda.is_available() else exit(1)"
```

Comment:
Are there any system commands which we can use to verify the existence of GPU instead of this ?

Reply:
I did a some research and seems no straightforward way. Any concern on the current method?